crates/graphql/src/schema/objects/marketplace.rs

```diff
@@ -74,8 +74,8 @@ impl Marketplace {
         &self.owner_address
     }
 
-    pub fn auction_house_address(&self) -> &str {
-        &self.auction_house_address
+    pub fn auction_house_address(&self) -> Option<&str> {
+        self.auction_house_address.as_deref()
     }
```

Comment:
Lets switch to `PublicKey<AuctionHouse>` fix up some of these string references.